### DISCUSSION TITLE
How can I notify users when they logged in ?

### DISCUSSION THREAD

**Discussion by 4Furki4** [nextauthjs/next-auth#8238] [Help]
Hello everyone, hope you are doing well.

I wonder how I can use some libraries such as react-toastify to notify users when they logged in successfully. I actually found a way to notify users but only with credential sign-in method.

````js
signIn('credentials', { ...userCrendentials, redirect: false }).then((res) => { // res is an object containing an error message, status and etc.
      if (res.error) {
        notify("Invalid credentials", "error")
      }
      else {
        notify("Logged in successfully", "success")
      }
})
````

However, couldn't implement this workaround in the signIn functions take providers such as google or github since the res is undefined. I read through the docs but found nothing.  

Is there a guide or resource that explains how to handle a situation like this? If not, what is the best way for me to notify the users?

**Comment by siddharthaasal** [comment]
Did you find any solution for this?
